recipes/monetio/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "monetio" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/noaa-oar-arl/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0bd6fc39d9ad7ab0899f5700fbe99715f87aedaf8edb50c5adaef36a984024fc
+
+build:
```

Comment:
can this be `noarch: python`? See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python